portfolio/src/main/webapp/functions/index.js

```diff
@@ -1,11 +1,26 @@
 const createCommentElement = commentPayload => {
     const username = document.createElement("span");
+    const sentiment = document.createElement("span");
     const comment = document.createElement("p");
     const deleteOption = document.createElement("i");
 
     username.innerHTML = `@ ${commentPayload.username}`;
     username.classList.add("badge", "badge-dark");
     comment.innerHTML = commentPayload.comment;
+
+    const score = parseFloat(commentPayload.sentiment);
+    const emotion =
+        score >= -1 && score <= -0.3 ? "Mad" :
```

Comment:
[super nit] how come these ones are capitalized, but the badges aren't?

Reply:
These ones are the actual text added to the HTML element, and the other ones are Bootstrap CSS classes. So their class naming is all lowercase, and the values "Happy", "Mad", "Neutral", are the textual value displayed.

Reply:
Ah, ok. Thanks for explaining!